website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -0,0 +1,118 @@
+---
+id: versioning-releases
+title:  ORCA Versioning and Releases
+description: Provides information to developers on semantic versioning and the release process.
+---
+
+Much of this documentation is also found at [Cumulus](https://github.com/nasa/cumulus/blob/master/docs/development/release.md).
+
+## Versioning
+
+The ORCA team uses semantic versioning. More information about semantic
+versioning can be found [here](https://semver.org/).
+
+## Release Process
+
+1. **Create a release branch.**
+
+From develop, create a new release branch from develop following the
+`release-MAJOR.MINOR.x`. For example, `release-1.14.1`. Push this branch 
+to github if you created it locally.
+
+2. **Update CHANGELOG.md.**
+
+Update the [CHANGELOG.md](https://github.com/nasa/cumulus-orca/blob/master/CHANGELOG.md). 
+Put a header under the 'Unreleased' section with the new version number and 
+the date.
+
+3. **Create a git tag for the release.**
+
+Make sure you're on the latest commit of the release branch.
+
+Create and push a new git tag:
+
+```
+    git tag -a vx.y.z -m "Release x.y.z"
+    git push origin vx.y.z
+```
+
+4. **Running the deployment**
+
+Publishing of new releases is handled by a Bamboo release plan and is manually
+triggered.
+
+If you created a new release plan in step one, you will need to create a new
+bamboo deployment plan.
+
+**Creating a Bamboo deployment plan**
+
+* In the ORCA project (https://ci.earthdata.nasa.gov/browse/ORCA-OI), click
+  `Actions -> Configure Plan` at the top right.
+* Scroll to the bottom of the branch list in the bottom left and select
+  `Create Plan Branch`.
+* Add the values in that list. Choose a display name that makes it very clear 
+  this is a deployment branch plan. Release (branch name) seems to work well. 
+  Make sure you enter the correct branch name.
+* Important Deselect Enable Branch - if you do not do this, it will immediately
+  fire off a build.
+* Do Immediately On the Branch Details page, enable 'Change Trigger'. Set the 

```

Comment:
Have to read this a couple times. 'Immediately, on the Branch Details page, enable 'Change Trigger'? What happens if you don't do it immediately?

Reply:
I'm not sure... maybe Brad will know.

Reply:
If not set to manual, any change (commits) to the release branch will trigger a build and stop at the release step. This is to help prevent unnecessary builds on the branch and having to wait. Ideally, we will move to a bamboo profile and away from the GUI someday soon so a lot of this will go away.

Reply:
Thanks for the info, Brad.